doc/api.md

```diff
@@ -35,122 +93,73 @@ Attribute | Type | Note
 `rwa_number` | `null` for BA61, string for BA80 | Essentially the internal bank account number
 `vendor` | string |
 
-### Proposal
-
-The central, generic data structure that maintains workflow information.
-
-Attribute | Type | Note
---- | --- | ---
-`steps` | [ [Step](#step) ] |
-`created_at` | string (time) |
-`id` | integer |
-`requester` | [User](#user) |
-`status` | string | Can be `pending`, `approved`, or `canceled`
-`updated_at` | string (time) |
-
-### User
-
-Attribute | Type | Note
---- | --- | ---
-`created_at` | string (time) |
-`email_address` | string | only shown when logged in
-`first_name` | string | only shown when logged in
-`id` | integer |
-`last_name` | string | only shown when logged in
-`updated_at` | string (time) |
-
 ## Endpoints
 
-### `GET /api/v1/users.json`
+### `GET /api/v2/proposals`
+
+Fetch one or more proposals matching a query. Default is all proposals to which you are subscribed.
 
 #### Query parameters
 
 All are optional.
 
 Name | Values
 --- | ---
-`limit` | an integer >= 0
-`offset` | an integer >= 0
+`size` | an integer >= 0
+`from` | an integer >= 0
+`limit` | alias for `size`
+`offset` | alias for `from`
+`status` | one of `pending`, `cancelled`, `complete`
+`start_date` | timestamp string
+`end_date` | timestamp.string
+`text` | full-text search string. See https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax
+
+#### Example
+
+```bash
+% curl -H 'Authorization: Bearer your-auth-token' \
```

Comment:
ah ok I saw

> include your Bearer auth token with every API request

and was thinking we'd see some type of auth token named `bearer` :)